python/src/trezorlib/cli/ethereum.py

```diff
@@ -252,18 +274,8 @@ def sign_tx(
     the connection with WEB3_PROVIDER_URI environment variable.
     """
 
-    # Web3 is used only in this one function, so importing it only here
-    try:
-        import web3
-    except ModuleNotFoundError:
-        click.echo("Ethereum requirements not installed.")
-        click.echo("Please run:")
-        click.echo()
-        click.echo("  pip install web3")
-        sys.exit(1)
-
     is_eip1559 = eip2718_type == 2
-    w3 = web3.Web3()
+    w3 = _get_web3()
```

Comment:
this way we still unconditionally require `web3` for the whole `sign_tx` function, even if we don't ever touch the instance.

instead, please replace all occurences of `w3` with `get_web3()`. This way the function `_erc20_contract` also doesn't need the `w3` instance passed in because it can also call `get_web3()`

Reply:
Done in [1933d73](https://github.com/trezor/trezor-firmware/pull/2124/commits/1933d73a34cdc39b7206a8f18a36f8daa1ce5e34)